toolkit/tools/grapher/grapher.go

```diff
@@ -35,10 +37,15 @@ func main() {
 
 	app.Version(exe.ToolkitVersion)
 	kingpin.MustParse(app.Parse(os.Args[1:]))
-
-	var err error
 	logger.InitBestEffort(*logFile, *logLevel)
 
+	prof, err := profile.StartProfiling(profFlags)
+	if err != nil {
+		logger.Log.Warnf("Could not start profiling: %s", err)
+		return
+	}
+	defer prof.StopProfiler()
+
```

Comment:
Do we want to make success here fully optional? This isn't a critical flow for the tools
```suggestion
	prof, err := profile.StartProfiling(profFlags)
	if err != nil {
		logger.Log.Warnf("Could not start profiling: %s", err)
	} else {
		defer prof.StopProfiler()
	}

```